### PR TITLE
Release notes for monitoring agent RPMs

### PR DESCRIPTION
**Pull request by agrawalkaushik**
Release notes for monitoring agent 1.1 and 2.0 releases.

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -119,6 +119,14 @@ module.exports = {
           "release_notes_128t_installer_2.1",
         ],
       },
+      {
+        "type": "category",
+        "label": "128T Monitoring Agent",
+        "items": [
+          "release_notes_monitoring_agent_1.1",
+          "release_notes_monitoring_agent_2.0",
```
Comment (MichaelBaj) [inline sidebars.js:127]: these should be in reverse order.  The newest should always be on top.

File: docs/release_notes_monitoring_agent_1.1.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+title: 128T Monitoring Agent 1.1 Release Notes
+sidebar_label: 1.1
+---
+# Release 1.1.0
```
Comment (MichaelBaj) [inline docs/release_notes_monitoring_agent_1.1.md:5]: headings should start with ## - as defined by the CONTRIBUTING.md#guidelines

File: docs/release_notes_monitoring_agent_1.1.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+# New Features and Improvements
```
Comment (MichaelBaj) [inline docs/release_notes_monitoring_agent_1.1.md:7]: should be ### h3

File: docs/release_notes_monitoring_agent_1.1.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+### MON-148 Top applications, sessions and sources input plugin
```
Comment (MichaelBaj) [inline docs/release_notes_monitoring_agent_1.1.md]: Following the pattern in other release notes these should be bulleted items just like ## Issues Fixed

File: docs/release_notes_monitoring_agent_1.1.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+The configuration for all 128T collectors such as t128_metrics, t128_events etc will automatically be staged in the inputs directory for convenience.
+
+### MON-171 Update Telegraf to latest stable version 1.14.0 ####
+A new stable version of telegraf was released upstream with several new inputs such as execd, wireguard and others.
+
+### MON-175 LTE metric collect will include SNR signal strength ####
+The `t128_lte_metric` collector will look for and report SNR signal strength if its reported by the 128T router.
+
+
+# Issues Fixed
```
Comment (MichaelBaj) [inline docs/release_notes_monitoring_agent_1.1.md:25]: This needs to be ## h2

File: docs/release_notes_monitoring_agent_1.1.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
+---
+title: 128T Monitoring Agent 1.1 Release Notes
+sidebar_label: 1.1
+---
+# Release 1.1.0
+
+# New Features and Improvements
+
+### MON-148 Top applications, sessions and sources input plugin
+To conflgure the new input plugin, please read the [Monitoring Agent Guide](plugin_monitoring_agent.md)
```
Comment (MichaelBaj) [inline docs/release_notes_monitoring_agent_1.1.md]: typo:  configure

Comment (ptimmons) [inline docs/release_notes_monitoring_agent_1.1.md]: configure

File: docs/release_notes_monitoring_agent_2.0.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
+---
+title: 128T Monitoring Agent 2.0 Release Notes
+sidebar_label: '2.0'
+---
+# Release 2.0.0
```
Comment (MichaelBaj) [inline docs/release_notes_monitoring_agent_2.0.md:5]: must be ## h2

File: docs/release_notes_monitoring_agent_2.0.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+# New Features and Improvements
```
Comment (MichaelBaj) [inline docs/release_notes_monitoring_agent_2.0.md:7]: Must be ## h2

File: docs/release_notes_monitoring_agent_2.0.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+### MON-148 Top applications, sessions and sources input plugin
+To conflgure the new input plugin, please read the [Monitoring Agent Guide](plugin_monitoring_agent.md)
+
+### MON-126 Automatically stage all 128T input configuration for easy of use ####
+The configuration for all 128T collectors such as t128_metrics, t128_events etc will automatically be staged in the inputs directory for convenience.
+
+### MON-171 Update Telegraf to latest stable version 1.14.0 ####
+A new stable version of telegraf was released upstream with several new inputs such as execd, wireguard and others.
+
+### MON-175 LTE metric collect will include SNR signal strength ####
```
Comment (MichaelBaj) [inline docs/release_notes_monitoring_agent_2.0.md]: Following the pattern in other release notes these should be bulleted items just like ## Issues Fixed

File: docs/release_notes_monitoring_agent_1.1.md [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+
+### MON-126 Automatically stage all 128T input configuration for easy of use ####
+The configuration for all 128T collectors such as t128_metrics, t128_events etc will automatically be staged in the inputs directory for convenience.
+
+### MON-171 Update Telegraf to latest stable version 1.14.0 ####
+A new stable version of telegraf was released upstream with several new inputs such as execd, wireguard and others.
+
+### MON-175 LTE metric collect will include SNR signal strength ####
+The `t128_lte_metric` collector will look for and report SNR signal strength if its reported by the 128T router.
```
Comment (ptimmons) [inline docs/release_notes_monitoring_agent_1.1.md:22]: "it is", not "its"

File: docs/release_notes_monitoring_agent_2.0.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+The `t128_lte_metric` collector will look for and report SNR signal strength if its reported by the 128T router.
+
+
+# Issues Fixed
+----
+- **MON-125** `t128_metrics` default bfd config doesn't work with 4.3
+
+  _**Resolution:**_ The new default config for metrics have the correct parameters for bfg metrics
```
Comment (MichaelBaj) [inline docs/release_notes_monitoring_agent_2.0.md:29]: typo: bfd metrics

File: docs/release_notes_monitoring_agent_2.0.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+----
+- **MON-146** Metric collector timing out with the default config on customer system
```
Comment (MichaelBaj) [inline docs/release_notes_monitoring_agent_2.0.md:31]: remove "on customer system"

File: docs/release_notes_monitoring_agent_1.1.md [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+
+
+# Issues Fixed
+
+------
+- **MON-146** Metric collector timing out with the default config on customer system
```
Comment (MichaelBaj) [inline docs/release_notes_monitoring_agent_1.1.md:28]: remove "on customer system"

File: docs/release_notes_monitoring_agent_1.1.md [hunk 14]
```diff
@@ -0,0 +1,39 @@
...
+
+  _**Resolution:**_ The metric configuration will now have a default timeout of 15 seconds.
+------
+- **MON-160** sample agent-config has invalid tags
+
+  _**Resolution:**_ All the sample configuration now contain valid data
```
Comment (ptimmons) [inline docs/release_notes_monitoring_agent_1.1.md:34]: "configurations"

File: docs/release_notes_monitoring_agent_1.1.md [hunk 15]
```diff
@@ -0,0 +1,39 @@
...
+------
+- **MON-169** peer-path collector only captures 1 peer-path per node
+
+  _**Resolution:**_ All peer paths on the node will be reported by the peer-path collector
+------
+- **MON-170** Default telegraf service (not 128T-telegraf) is enabled and running un-necessarily on the system
```
Comment (ptimmons) [inline docs/release_notes_monitoring_agent_1.1.md:40]: unnecessarily

File: docs/release_notes_monitoring_agent_2.0.md [hunk 16]
```diff
@@ -0,0 +1,42 @@
+---
+title: 128T Monitoring Agent 2.0 Release Notes
+sidebar_label: '2.0'
+---
+# Release 2.0.0
+
+# New Features and Improvements
+
+### MON-148 Top applications, sessions and sources input plugin
+To conflgure the new input plugin, please read the [Monitoring Agent Guide](plugin_monitoring_agent.md)
```
Comment (ptimmons) [inline docs/release_notes_monitoring_agent_2.0.md]: configure

File: docs/release_notes_monitoring_agent_2.0.md (hunk 11 above)
Comment (ptimmons) [inline docs/release_notes_monitoring_agent_2.0.md:29]: BFD, not "bfg"

File: docs/release_notes_monitoring_agent_2.0.md [hunk 17]
```diff
@@ -0,0 +1,42 @@
...
+
+  _**Resolution:**_ The new default config for metrics have the correct parameters for bfg metrics
+----
+- **MON-146** Metric collector timing out with the default config on customer system
+
+  _**Resolution:**_ The metric configuration will now have a default timeout of 15 seconds.
+------
+- **MON-160** sample agent-config has invalid tags
+
+  _**Resolution:**_ All the sample configuration now contain valid data
```
Comment (ptimmons) [inline docs/release_notes_monitoring_agent_2.0.md:37]: configurations

File: docs/release_notes_monitoring_agent_2.0.md [hunk 18]
```diff
@@ -0,0 +1,42 @@
...
+------
+- **MON-169** peer-path collector only captures 1 peer-path per node
+
+  _**Resolution:**_ All peer paths on the node will be reported by the peer-path collector
+------
+- **MON-170** Default telegraf service (not 128T-telegraf) is enabled and running un-necessarily on the system
```
Comment (ptimmons) [inline docs/release_notes_monitoring_agent_2.0.md:43]: unnecessarily

**Comment by MichaelBaj** [comment]
Also, I added integration between Jira and GitHub so you should be able to use smart commit hooks in your comments to associate this with the corresponding Jira issue ID.

File: docs/release_notes_monitoring_agent_1.1.md (hunk 4 above)
Comment (agrawalkaushik) [reply docs/release_notes_monitoring_agent_1.1.md]: I wanted to try this as the features get better highlighted this way. I felt it improves the readability a bit

File: docs/release_notes_monitoring_agent_1.1.md (hunk 6 above)
Comment (agrawalkaushik) [reply docs/release_notes_monitoring_agent_1.1.md]: Fixed

**Comment by agrawalkaushik** [comment]
> Also, I added integration between Jira and GitHub so you should be able to use smart commit hooks in your comments to associate this with the corresponding Jira issue ID.

That is cool - I just created a task and added the JIRA id to my latest commit.

**Comment by MichaelBaj** [comment]
Kaushik addressed PT's changed in his latest commit.

**Pull request merged**